dist/.magento.env.yaml

```diff
@@ -262,6 +262,27 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTICSUITE_CONFIGURATION - use this variable to retain customized ElasticSearch service settings                  #
+# between deployments. Is added under 'system/default/smile_elasticsuite_core_base_settings' system configuration.    #
+# Magento Version: 2.2.0 and later                                                                                    #
```

Comment:
Doesn't ElasticSuite work with 2.1.x?

Reply:
According to their documentation - no
https://github.com/Smile-SA/elasticsuite/wiki/ModuleInstall#requirements